packages/protocol/contracts/stability/GrandaMento.sol

```diff
@@ -152,32 +197,167 @@ contract GrandaMento is
     IERC20 sellToken = sellCelo ? getGoldToken() : IERC20(stableToken);
     require(
       sellToken.transferFrom(msg.sender, address(this), sellAmount),
-      "Transfer of sell token failed"
+      "Transfer in of sell token failed"
     );
 
     // Record the proposal.
-    uint256 proposalId = exchangeProposalCount;
-    exchangeProposals[proposalId] = ExchangeProposal({
+    // Add 1 to the running proposal count and use as the proposalId. This is
+    // intentionally never 0 because LinkedList, which is used for keeping track
```

Comment:
Technically, it's 0 when there hasn't been any exchange proposal created, right?